### PR TITLE
Use case post on deleting call recordings / company call recordings b…

### PR DESCRIPTION
**Pull request by tbass134**
Use case post on deleting call recordings / company call recordings by duration of call

### CODE REVIEW

File: _use_cases/en/delete_recordings_by_duration.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+In this example, we will query for the last month of recordings and delete calls that are less than 30 seconds in length. We'll be using a python script to query for the call recordings and for each recording, we'll delete the recording.
+
+Python is the language used to build the following example; however, you can use any language with which you are most comfortable. The following Python library is used in this example:
+
+* [requests](https://requests.readthedocs.io/en/master/)
+
+## Prerequisites
+
+Before you can get started, you will need to have a Vonage Developer account. If you do not have a Vonage Developer account, please use this [guide](/concepts/guides/create-a-developer-account) to setup and create your account.
```
Comment (niati) [inline _use_cases/en/delete_recordings_by_duration.md:23]: link broken...should be: /getting-started/create-a-developer-account

File: _use_cases/en/delete_recordings_by_duration.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+After you have an account, you will need to do the following using these guides:
+
+* [Create an application](/concepts/guides/create-an-application).
```
Comment (niati) [inline _use_cases/en/delete_recordings_by_duration.md:27]: Links broken...should be:

/getting-started/create-an-application

/getting-started/subscribe-to-apis

File: _use_cases/en/delete_recordings_by_duration.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+* [Subscribe to the API's](/concepts/guides/subscribe-to-apis).
+
+For this example, you will need to Subscribe to the [Call Recording API](/call-recording/overview).
+
+## Authentication
+
+After creating an application and subscribing to the Call Recording API, you will now need to log-in using your Vonage Business Cloud credentials. This example application uses the Requests library to call the `/api/accounts/` API. Check out the [Making an API Request guide](/concepts/guides/make-an-api-request) for more details.
```
Comment (niati) [inline _use_cases/en/delete_recordings_by_duration.md:34]: Link broken...should be:

/getting-started/make-an-api-request

**Comment by niati** [comment]
@tbass134 just a few minor changes requested...see above.

**Comment by tbass134** [comment]
thanks @niati for going over this, i've updated the links

**Pull request merged**